magento/magento2
# #39382: Fix incorrect image display for directories with identical names but different case

### Description (*)
This pull request addresses an issue where files uploaded to a specific directory in the Media Gallery are also visible in directories with similar names but different case.

### Fixed Issues (if relevant)
1. Fixes magento/magento2#39382

### Manual testing scenarios (*)
1. OS - Ubuntu
2. Navigate to Content > Media Gallery
3. Create a folder under wysiwyg "Testing"
4. Upload an image in the newly created folder.
5. Select wysiwyg folder and try to create another folder named "testing"
6. Image uploaded to "Testing" folder should not show in "testing" folder.


### Questions or comments


### Contribution checklist (*)
